foamliu_i-Cloud_utils_face_128.py

```
@@ -0,0 +1,456 @@
+    return num_faces, float(elapsed), bboxes, landmarks
+
+
+def face_search():
+    start = time.time()
+    ensure_folder(STATIC_DIR)
+    ensure_folder(UPLOAD_DIR)
+    file = request.files['file']
+    filename = secure_filename(file.filename)
+    filename = filename.lower()
+    if filename in ['jpg', 'jpeg', 'png', 'gif']:
+        filename = str(random.randint(0, 101)) + '.' + filename
```

Ruff (0.13.1)

Standard pseudo-random generators are not suitable for cryptographic purposes

(S311)